config/k8s-demo.yml

```diff
@@ -29,17 +29,17 @@ cluster_groups:
 - "{{ cluster_worker_group }}"
```

Comment:
Can we just remove this file since your demo is over?

Reply:
Can we leave it here a wee bit longer. I'll deal with it in a separate PR.

Reply:
Sure